src/ui/src/app/dashboard/dashboard-body/main/leave-management/add-new-leave-type/add-new-leave-type.component.ts

```diff
@@ -3,7 +3,7 @@ import { FormBuilder, FormControl, FormGroup, FormsModule, NgSelectOption, React
 import { IAddNewLeaveType } from '../../../../../interface/Add-New-Leave-Type-Interface';
 import { CommonModule } from '@angular/common';
 import { NgSelectComponent } from '@ng-select/ng-select';
-import { LeaveManagementService } from '../../../../../service/leave-management.service';
+import { LeaveManagementService } from '../../../../../Service/leave-management.service';
```

Comment:
folder name should be in small letter

Reply:
having some common issue regarding the names of folders in the interface and service folder due to which we were not able to update it